### ISSUE TITLE
Not found: No registered 'Pad' OpKernel for CPU devices compatible with node Pad_12 = Pad[T=DT_STRING, Tpaddings=DT_INT32](Reshape_6, stack_12)

### ISSUE THREAD

**Issue by xinyuegtxy** [tensorflow/tensorflow#21084]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
- **TensorFlow installed from (source or binary)**:binary
- **TensorFlow version (use command below)**:1.5
- **Python version**:2.7
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:5.4
- **CUDA/cuDNN version**: cuda9.0 cudnn 7 nvidia- 384driver 
- **GPU model and memory**: 12G
- **Exact command to reproduce**:
python object_detection/model_main.py     --pipeline_config_path=${PIPELINE_CONFIG_PATH}     --model_dir=${MODEL_DIR}     --num_train_steps=${NUM_TRAIN_STEPS}     --num_eval_steps=${NUM_EVAL_STEPS}     --alsologtostderr
 TITANX  python object_detection/model_main.py \
>     --pipeline_config_path=${PIPELINE_CONFIG_PATH} \
>     --model_dir=${MODEL_DIR} \
>     --num_train_steps=${NUM_TRAIN_STEPS} \
>     --num_eval_steps=${NUM_EVAL_STEPS} \
>     --alsologtostderr
WARNING:tensorflow:Estimator's model_fn (<function model_fn at 0x7eff51d9a6e0>) includes params argument, but params are not passed to Estimator.
WARNING:tensorflow:num_readers has been reduced to 1 to match input file shards.
WARNING:tensorflow:From /mnt/object_detection/core/box_predictor.py:407: calling reduce_mean (from tensorflow.python.ops.math_ops) with keep_dims is deprecated and will be removed in a future version.
Instructions for updating:
keep_dims is deprecated, use keepdims instead
WARNING:tensorflow:From /mnt/object_detection/meta_architectures/faster_rcnn_meta_arch.py:2037: get_or_create_global_step (from tensorflow.contrib.framework.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Please switch to tf.train.get_or_create_global_step
WARNING:tensorflow:From /mnt/object_detection/core/losses.py:317: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
Instructions for updating:

Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

See tf.nn.softmax_cross_entropy_with_logits_v2.

WARNING:tensorflow:From /mnt/object_detection/meta_architectures/faster_rcnn_meta_arch.py:1930: calling reduce_sum (from tensorflow.python.ops.math_ops) with keep_dims is deprecated and will be removed in a future version.
Instructions for updating:
keep_dims is deprecated, use keepdims instead
/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/gradients_impl.py:97: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
2018-07-24 16:38:16.076136: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-07-24 16:38:16.529363: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:895] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2018-07-24 16:38:16.532926: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1105] Found device 0 with properties: 
name: TITAN X (Pascal) major: 6 minor: 1 memoryClockRate(GHz): 1.531
pciBusID: 0000:00:06.0
totalMemory: 11.90GiB freeMemory: 11.75GiB
2018-07-24 16:38:16.532987: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1195] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: TITAN X (Pascal), pci bus id: 0000:00:06.0, compute capability: 6.1)
2018-07-24 16:38:34.429255: E tensorflow/core/common_runtime/executor.cc:651] Executor failed to create kernel. Not found: No registered 'Pad' OpKernel for CPU devices compatible with node Pad_12 = Pad[T=DT_STRING, Tpaddings=DT_INT32](Reshape_6, stack_12)
         (OpKernel was found, but attributes didn't match)
        .  Registered:  device='GPU'; T in [DT_INT32]; Tpaddings in [DT_INT64]
  device='GPU'; T in [DT_INT32]; Tpaddings in [DT_INT32]
  device='GPU'; T in [DT_DOUBLE]; Tpaddings in [DT_INT64]
  device='GPU'; T in [DT_DOUBLE]; Tpaddings in [DT_INT32]
  device='GPU'; T in [DT_FLOAT]; Tpaddings in [DT_INT64]
  device='GPU'; T in [DT_FLOAT]; Tpaddings in [DT_INT32]
  device='GPU'; T in [DT_HALF]; Tpaddings in [DT_INT64]
  device='GPU'; T in [DT_HALF]; Tpaddings in [DT_INT32]
  device='CPU'; T in [DT_BOOL]; Tpaddings in [DT_INT64]
  device='CPU'; T in [DT_BOOL]; Tpaddings in [DT_INT32]
  device='CPU'; T in [DT_COMPLEX128]; Tpaddings in [DT_INT64]
  device='CPU'; T in [DT_COMPLEX128]; Tpaddings in [DT_INT32]
  device='CPU'; T in [DT_COMPLEX64]; Tpaddings in [DT_INT64]

**Comment by miibotree** [comment]
got the same error.

**Comment by JinYue2015** [comment]
Hi，have you solved the problem yet? I have the same problem

**Comment by facaiy** [comment]
Could you take a test on the latest version of tensorflow?

**Comment by johnlinml** [comment]
I had the same issue. However, after I updated tensorflow from 1.5.0 to 1.9.0. It runs smoothly.

**Comment by bignamehyp** [comment]
@xinyuegtxy does this issue resolve after up-grading to the latest tensorflow?

**Comment by johnlinml** [comment]
To me, upgrading tensorflow from 1.5.0 to 1.9.0 resolves the issue.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 30 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by xinyuegtxy** [comment]
Tf1.9 resolve it

**Comment by upwon** [comment]
tensorflow 1.11 or 1.9 is ok.
